### DISCUSSION TITLE
📚 document: Which qbittorrent ( + other client ) options are compatible with flood, and which need to be disabled?

### DISCUSSION THREAD

**Discussion by C84186** [jesec/flood#222] [General]
Type: Documentation Issue

## Summary

I've noticed that in a few places certain client features are flagged as unsupported - For example, qbittorrent's file management (Apparently flood implements its own management).

In the docs, there's not really a comprehensive list in any place on what flood does not support, and particularly, what flood does not intend to support by design.

**Comment by jesec** [comment]
This is not necessary in my opinion. You simply use Flood and you don't need to care about the underlying client and its features.

Flood does have some features that are supported by some clients but not others. For instance, sequential downloading. A feature list API, which allows frontend to hide unavailable features, is on the roadmap. 

Regardless, wiki is open to contributions. You may add a page if you feel that others cam benefit from such a list. I am moving this to discussions.

**Reply by C84186** [reply]
I understand what you're saying, but I'd also like to understand which features are definitely-avoid for flood, & which aren't. 

For example, I haven't disabled qbt's automatic save management, but I don't know if that will break things
